dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/DescribeStep.java

```diff
@@ -64,9 +70,19 @@ public DataDescribe getData(SubmissionService submissionService, InProgressSubmi
 
     private void readField(InProgressSubmission obj, SubmissionStepConfig config, DataDescribe data,
                            DCInputSet inputConfig) throws DCInputsReaderException {
+        String documentTypeValue = "";
+        List<MetadataValue> documentType = itemService.getMetadataByMetadataString(obj.getItem(),
+                configurationService.getProperty("submit.type-bind.field", "dc.type"));
+        if (documentType.size() > 0) {
```

Comment:
empty() !?

Reply:
@vidiecan  This is not my code, I copied it from the upstream